docs/rest_api.md

```diff
@@ -34,7 +34,7 @@ These are the collection endpoints:
 - `https://{APIHOST}/api/v1/namespaces/{namespace}/packages`
 - `https://{APIHOST}/api/v1/namespaces/{namespace}/activations`
 
-The `{APIHOST}` is the OpenWhisk API hostname (for example, openwhisk.ng.bluemix.net, 172.17.0.1, and so on).
+The `{APIHOST}` is the OpenWhisk API hostname (for example, localhost, 172.17.0.1, and so on).
```

Comment:
@markusthoemmes notice here how APIHOST is introduced -- i preserved this in the early section.
Maybe this just confusing and I should make them all $APIHOST instead. Thoughts?